### ISSUE TITLE
Improve Exclamation Point

### ISSUE THREAD

**Issue by jsjoeio** [excalidraw/excalidraw#381]
First, this project is awesome!

Using the text element, I noticed an exclamation point "!" and the letter "l" are hard to differentiate between.

<img width="103" alt="image" src="https://user-images.githubusercontent.com/3806031/72487566-b010c800-37cb-11ea-9973-d6e7a4296210.png">

What if there the exclamation point were redrawn so that it looked more distinct compared to the "l"?

### Follow-up Questions
- Would this be a good first issue to take on?
- Where does this even live in the codebase? (I took a quick look but couldn't figure it out. I can keep looking but thought I'd ask :)

**Comment by vjeux** [comment]
I agree! We’re using the Virgil font that has this behavior. If you’re able to tweak that particular symbol in the font that would be awesome!

**Comment by vjeux** [comment]
https://github.com/excalidraw/excalidraw/blob/master/public/FG_Virgil.ttf

**Comment by giovannigiordano** [comment]
@vjeux we can even change the font?

**Comment by jsjoeio** [comment]
Awesome! I've never done that before but I'm up for the challenge.

(Notes for myself - todos):
- [ ] Find font editor and edit "!" symbol (maybe [birdfont.org](https://birdfont.org/))
- [ ] Fork excalidraw and test out locally
- [ ] Open PR

**Comment by vjeux** [comment]
I’ve never edited a font either, but that sounds like a cool challenge, let me know how it goes!

**Comment by jsjoeio** [comment]
The hardest part was finding a good font editor 😂 I ended up using [Glyphs](https://glyphsapp.com/) (free trial). 

To actually accomplish this, I had to trace the bottom part of the glyph and add a new shape, then bring the top part of the glyph up. Then I exported it and viola! 

In my research, I also found a free online too for making your own font called [Glyphter](https://glyphter.com/) (worth bookmarking).
